io_handler.rb

```diff
@@ -0,0 +1,15 @@
+class IOHandler
+
+  def printer printable_item
+    puts printable_item
+  end
+
+  def get_input
+    gets.chomp
+  end
+
+  def word_blank word
```

Comment:
Can't decide if this method belongs here or in the Prompter... Something to think about! Curious what @NicoleCarpenter thinks